bump_version.sh

```diff
@@ -20,15 +20,15 @@ else
         major|minor|patch|prerelease|build)
             new_version=$(python -c "import semver; print(semver.bump_$1('$old_version'))")
             echo Changing version from "$old_version" to "$new_version"
-            sed -i "s/$old_version/$new_version/" $VERSION_FILE
+            sed "s/$old_version/$new_version/" $VERSION_FILE > >(sleep 1 && cat > $VERSION_FILE)
```

Comment:
I don't think we should use process substitution for the sake of using process substitution.

This seems much clearer and less hacky.
```
sed "s/$old_version/$new_version/" $VERSION_FILE > /tmp/version.$$
cp /tmp/version.$$ $VERSION_FILE
```

Reply:
mv. yes. 👍

Reply:
Did we see this as well? https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

Reply:
You can do it that way but you need to delete the .bak file afterwards. It's really the same thing.

Reply:
@dav3r , choose and perish.

Reply:
As usual, @jsf9k is correct.  His solution is cleaner, clearer, and more robust.  So that's what I went with in c3b0794.